react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -0,0 +1,40 @@
+import { isArray } from './utils';
+
+export function min(...numbers) {
+  if (numbers.length === 0) {
+    return undefined;
+  }
+
+  if (numbers.length === 1) {
+    if (isArray(numbers[0])) {
+      return Math.min(...numbers[0]);
+    }
+
+    return numbers[0];
+  }
+
+  return Math.min(...numbers);
+}
+
+export function copy(myObject) {
+  if (isArray(myObject)) {
+    return myObject.slice();
+  }
+
+  return { ...myObject };
+}
+
+export function reverseMerge(myArray1, myArray2) {
+  const myReversedArray = copy(myArray2);
```

Comment:
I don't think using the `copy` method is necessary here

Reply:
I'm using `copy` because a I need a _copy_ of the array.
If I use `const myReversedArray = myArray2;` the array is passed by reference and then the test `reverseMerge does not mutate the arguments` don't comply.
Or do you mean to use another _copy_ method?

Reply:
yes, you can create a new array like this `[something, someOtherThing]`